.github/workflows/test.yaml

```diff
@@ -169,6 +198,29 @@ jobs:
           # TODO add arch?
           - fedora:latest
           - opensuse/tumbleweed
+  
+  # The required-unit-tests job was introduced to maintain a consistent
+  # status check name, regardless of changes to the Go channel used for unit
+  # tests. This avoids the need to update required status checks whenever the
+  # Go channel changes.
+  required-unit-tests:
+    runs-on: ubuntu-latest
+    needs:
+      - unit-tests
+      - unit-tests-special
+      - unit-tests-cross-distro
+    if: always()
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          path: ./src/github.com/snapcore/snapd
```

Comment:
Same as other comment, there is no reason to add the fetch-depth or a custom path

Reply:
Thanks, done